tests/testthat/test_XcmsExperiment.R

```diff
@@ -1326,3 +1315,11 @@ test_that("setAs,XcmsExperiment,xcmsSet works", {
     expect_s4_class(res, "xcmsSet")
     expect_equal(peaks(res), chromPeaks(xmseg))
 })
+
+test_that("storeResults,RDataParam works", {
+    param <- RDataParam(fileName ="test")
+    param2 <- RDataParam()
+    expect_false(is.null(param2))
+    storeResults(xmse, param = param)
+    expect_true(file.exists("test"))
+})
```

Comment:
Do you think about any other ? 
I was thinking i could also try to load() it and see its class, but maybe this is overkill.

Reply:
Better to test more than less. I would actually load it again and see if the objects are the same.